### PR TITLE
fix Wardform flickering

### LINKED ISSUE #548: Ward Form Flickering Error
### Describe the bug

Ward Form Flickering for the name, shortly showing an input error

### Describe how to produce the bug

load the ward details

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #548

### CODE REVIEW

File: tasks/components/WardForm.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ export const WardForm = ({
     onChange(newWard, isValid)
   }
 
+  console.log(ward)
```
Comment (use-to) [inline tasks/components/WardForm.tsx:90]: Intended?

File: tasks/components/layout/WardDetails.tsx [hunk 2]
```diff
@@ -109,8 +108,8 @@ export const WardDetail = ({
   return (
     <div className={tw('flex flex-col py-4 px-6')}>
       <LoadingAndErrorComponent
-        isLoading={isLoading && !!ward}
-        hasError={isError && !!ward}
+        isLoading={(isLoading && !isCreatingNewWard && !ward) || (!isCreatingNewWard && !newWard.id)}
```
Comment (use-to) [inline tasks/components/layout/WardDetails.tsx:111]:
```suggestion
        isLoading={!isCreatingNewWard && ((isLoading && !ward) ||  !newWard.id)}
```

**Review by use-to — CHANGES_REQUESTED** [review]

File: tasks/components/WardForm.tsx (hunk 1 above)
Comment (DasProffi) [reply tasks/components/WardForm.tsx:90]: No, just forgot it

**Review by use-to — APPROVED** [review]
LGTM

**Pull request merged**